keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -680,9 +744,9 @@ by using the feature of [job tracking with finalizers](https://kubernetes.io/doc
 
 #### Evolving condition types
```

Comment:
What about adding/removing reasons within DisruptionTarget/ResourceLimitExceeded? Will it be considered a non-breaking change? (perhaps it can, but we should document that users should rely on generic sense of "disrupted for cluster management" rather than specifics of current reasons)

Reply:
Currently there is no API to match pod conditions based on the reason field. The reasons listed in "Design details" are only to gain consensus on naming them for the implementation purposes. See the added paragraph in the "Design details" section.